### DISCUSSION TITLE
BaseURL as http and https

### DISCUSSION THREAD

**Discussion by Ncoder2** [projectdiscovery/nuclei#488] [Q&A] [answered]
Hello,

The URL list to be scanned sometimes comes with http or https tag or just the domain. 
So if all the URLs in a large list contain only the domain names like google.com, amazon.com, is there a way to tell nuclei to scan both http and https of the baseURL ?

**Comment by ehsandeep** [comment, marked as answer]
Hey @Ncoder2,

Nuclei accept URL as an input to run HTTP based templates. 

> is there a way to tell nuclei to scan both http and https of the baseURL ?

No this is not supported as we have dedicated projects to do this, you can simply put [httpx](https://github.com/projectdiscovery/httpx) in between which will probe the URLs and pass them to nuclei to process it.

```
httpx -l subs.txt | nuclei -t tempaltes_to_run
```

If you want to run nuclei on both HTTP and HTTPS versions of the website you can do it using the `no-fallback` flag in `httpx`.

```
httpx -l subs.txt -no-fallback | nuclei -t tempaltes_to_run
```

**Reply by Ncoder2** [reply]
Thank you @bauthard ! This helps !
